### ISSUE TITLE
False positive report of aria-hidden-focus 

### ISSUE THREAD

**Issue by EqualEntry-Heather** [microsoft/accessibility-insights-web#3371]
## Describe the bug

Ran MS A11y Insights for Web on a clients test environment and results appear to be a false positive...
Failed - aria-hidden-focus: Ensures aria-hidden elements do not contain focusable elements

Snippet provide = 
 `<div data-index="-1" tabindex="-1"   class="slick-slide slick-cloned" aria-hidden="true"   style="width: 1338px;">`

How to fix Focusable content should have tabindex='-1' or   be removed from the DOM

Code snippet provided shows that it does have a tabindex="-1"

Will admit that something is wrong with their carousel as there are tab stops that a keyboard users is unaware of where the focus is.

**To Reproduce**
test environment is password protected and I don't have authorization to provide a user name and password to repro exactly but believe the Export from MSA11yInsightsWeb will be enough.

## Expected behavior

Insight should flag errors that are errors and point to fixes that aren't already implemented.

## Screen shot
<img width="954" alt="bugreport" src="https://user-images.githubusercontent.com/43451139/93506537-66442880-f8ea-11ea-8f35-8160c845a3d6.png">

## Context (please complete the following information)

- OS Name & Version: Windows 10 Home Version 2004, build 19041.508 Windows Feature Experience Pack 120.2212.31.0
- Browser Version: Google Chrome Version 85.0.4183.102 (Official Build) (64-bit)
- Target Page: https://moco-508.demo.socrata.com/#content

## Are you willing to submit a PR?

no

## Did you search for similar existing issues?
yes

## Additional context

Was unable to attached MS Insights report export - error "We don't support that file type."

**Comment by lisli1** [comment]
Hi @EqualEntry-Heather, I tried to repro this [here](https://codepen.io/lisli1/pen/MWyPjrV) with the code snippet you provided, but I did not get an error. Based on this and the tab stop issue you mentioned, I suspect that something else or a combination of things on the page is causing this issue.
